src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Microsoft.Health.Fhir.TemplateManagement.UnitTests.csproj

```diff
@@ -17,14 +17,18 @@
     </PackageReference>
   </ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\Microsoft.Health.Fhir.TemplateManagement\Microsoft.Health.Fhir.TemplateManagement.csproj" />
-  </ItemGroup>
-
   <ItemGroup>
     <None Update="TestData\**">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="TestData\UserFolder\" />
+  </ItemGroup>
```

Comment:
We have already include the `TestData` folder in Line 21. Can we remove this item group here?

Reply:
Thanks, fixed it.